lib/bindConnectors/bindMessage.js

```diff
@@ -21,6 +21,18 @@ module.exports = function (message, threadneedle, connectorConfig) {
         methodName = '#' + methodName;
     }
 
+    //Intercept afterSuccess
+    if (_.isObject(message.model)) {
```

Comment:
Functions are technically objects, so this line won't work as expected.

Trying running `_.isObject(function () {})` in Lodash - it'll return true.